.github/workflows/build-and-test.yml

```diff
@@ -110,7 +110,7 @@ jobs:
       - name: CheckDoc
         run: make checkdoc
       - name: Porto
-        run: make porto
```

Comment:
I believe `make porto` runs recursively so no need to run it in every module. We can probably remove the goporto target

Reply:
Tested. You are correct. I removed the `make porto` rule from every target and kept the one that updates all.